packages/bits/src/lib/image/image.component.ts

```diff
@@ -5,8 +5,10 @@ import {
     Component,
     ElementRef,
     Inject,
-    Input,
-    OnInit,
+    Input, 
+    OnChanges,
+    OnInit, 
```

Comment:
I think lines 8 and 10 have a trailing space which is causing the build to fail here due to lint errors: https://app.circleci.com/pipelines/github/solarwinds/nova/1690/workflows/f8485c15-1382-4984-a2f4-0bd1e2f57708/jobs/13742

Reply:
Thank you. I already found it